client/components/post-list-fetcher/index.jsx

```diff
@@ -1,21 +1,21 @@
 /**
  * External dependencies
  */
-var React = require( 'react' );
+const React = require( 'react' );
```

Comment:
Is there a good reason to migrate these to use `let` / `const`, but keep using `require` instead of `import`? Perhaps you just want to satisfy the linter?